### PR TITLE
[Integ] Setup userArn for messaging test for prev version

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
- Add a step to setup userArn before running messaging session test in previous version integration test.]
- Readjust the cron to run once every day at 1am.

**Testing**

1. Have you successfully run `npm run build:release` locally? N/A
2. How did you test these changes? Switch over to PR trigger and verify that messaging session completed successfully. 
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/script/get-prev-version [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+
+const versions = exec('npm view amazon-chime-sdk-js versions --json').toString().trim().split("\n");
+// The output would be something like
+// [
+//   "1.0.0",
+//       ...
+//   "2.8.0", <- previous version
+//   "2.9.0"  <- latest version
+// ]
+// exec('npm view amazon-chime-sdk-js versions --json', (err, stdout, _) => {
```
Comment (devalevenkatesh) [inline .github/script/get-prev-version:14]: Want to remove these comments :) and the ruby one below?

File: .github/workflows/prev-version-integration.yaml [hunk 2]
```diff
@@ -2,14 +2,16 @@ name: Previous Version Integration Tests Workflow
...
+    # More information on cron https://crontab.guru/
+    - cron: '0 1 * * *'
 
 env:
- SELENIUM_GRID_PROVIDER: saucelabs
- CLOUD_WATCH_METRIC: false
- TEST_TYPE: Github-Action
- SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
- SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+  SELENIUM_GRID_PROVIDER: saucelabs
```
Comment (devalevenkatesh) [inline .github/workflows/prev-version-integration.yaml:9]: Indentation seems off, just note to check once.

Comment (ltrung) [reply .github/workflows/prev-version-integration.yaml:9]: I think one or 2 spaces are the same. Change to 2 to be more consistent with other places.

File: .github/script/get-prev-version (hunk 1 above)
Comment (ltrung) [reply .github/script/get-prev-version]: Oops :D

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**